--- astyle-1.15.3.orig/Makefile	2006-05-12 15:49:17.000000000 +0200
+++ astyle-1.15.3/Makefile	2006-05-12 15:49:37.000000000 +0200
@@ -4,10 +4,10 @@
 OBJS = ASResource.o ASBeautifier.o ASFormatter.o astyle_main.o
 
 astyle: $(OBJS)
-	g++ $(CPPFLAGS) -o astyle  $(OBJS)
+	$(CXX) $(CPPFLAGS) -o astyle  $(OBJS)
 
 .cpp.o:
-	g++ $(CPPFLAGS) -c $<
+	$(CXX) $(CPPFLAGS) -c $<
 .SUFFIXES: .cpp .c .o
 
 clean:
